frontend/public/components/catalog/catalog-items.tsx

```diff
@@ -186,33 +266,32 @@ export class CatalogTileViewPage extends React.Component {
     this.renderTile = this.renderTile.bind(this);
   }
 
-  componentDidMount() {
+  componentDidMount(): void {
     const { items } = this.props;
-    const searchParams = new URLSearchParams(window.location.search);
-    const detailsItemID = searchParams.get('details-item');
-    const detailsItem =
+    const searchParams: URLSearchParams = new URLSearchParams(window.location.search);
+    const detailsItemID: string = searchParams.get('details-item');
+    const detailsItem: Item =
       detailsItemID && _.find(items, (item) => detailsItemID === _.get(item, 'obj.metadata.uid'));
-
     this.setState({ detailsItem });
   }
 
-  openOverlay(detailsItem) {
-    const params = new URLSearchParams(window.location.search);
+  openOverlay(detailsItem: Item): void {
```

Comment:
please write it like :
```suggestion
  openOverlay = (detailsItem: Item): void => {
```
and get rid of `this.openOverlay = this.openOverlay.bind(this);`

Reply:
changed

Reply:
done